includes/abstracts/abstract-wc-stripe-payment-gateway.php

```diff
@@ -1322,7 +1322,8 @@ public function confirm_intent( $intent, $order, $prepared_source ) {
 
 		// Try to confirm the intent & capture the charge (if 3DS is not required).
 		$confirm_request = [
-			'source' => $prepared_source->source,
+			'source'     => $prepared_source->source,
+			'return_url' => $this->get_return_url( $order ),
```

Comment:
I think this might be wrong; we should only include the `return_url` parameter when `confirm` is set to `true` 🤔 

See [the docs](https://stripe.com/docs/api/payment_intents/create#create_payment_intent-return_url)